features/step_definitions/scmpuff_steps.rb

```diff
@@ -73,13 +73,13 @@
       system("git commit -m.")
     end
   end
-  FileUtils.cp_r MOCK, current_dir
+  FileUtils.cp_r MOCK, expand_path(".")
```

Comment:
This is a silly and outdated rule, Hound, but I don't care enough to change your settings right now.